protocol/v2/ssv/runner/aggregator.go

```diff
@@ -124,6 +143,7 @@ func (r *AggregatorRunner) ProcessPreConsensus(logger *zap.Logger, signedMsg *sp
 		DataSSZ: byts,
 	}
 
+	r.metrics.StartConsensus()
 	if err := r.BaseRunner.decide(logger, r, duty.Slot, input); err != nil {
 		return errors.Wrap(err, "can't start new duty runner instance for duty")
 	}
```

Comment:
Seems `r.metrics.StartConsensus()` call have been missing here ? I see it made in other runners, doesn't look like this is different.